index.html

```diff
@@ -6,51 +6,48 @@
 <meta name=description content="Subpicture is a photo project by Ryan Van Etten currently in San Francisco">
 <meta name=author content="Ryan Van Etten">
 <style>
-  ::-moz-selection { background: var(--bee); color: var(--nite) }
-       ::selection { background: var(--bee); color: var(--nite) }
-  :root {
-    --bee: #ee0;
-    --nite: #111;
-    --lite: #efefef;
-    background-color: var(--nite);
-    color: var(--lite);
-  }
+::-moz-selection { background: springgreen; color: black }
+     ::selection { background: springgreen; color: black }
+
+ :root {
+  --nite: #111;
+  --lite: #efefef;
+ }
+
+ @supports (--live:1) {
+   :root {
+      background: var(--nite);
+      color: var(--lite);
+   }
+ }
+
   html {
+    background: #111;
+    color: #efefef;
     font-family: sans-serif;
     -webkit-text-size-adjust: 100%;
-    -ms-text-size-adjust: 100%;
   }
+
   body {
     margin: 0 auto;
     font-size: 1.25em;
     line-height: 1.4;
     padding: 2em;
     max-width: 960px;
   }
-  svg {
-    max-width: 100%;
-    height: auto;
-  }
-  .m-auto { margin: auto }
-  .m-duct { margin-top: 1rem }
-  .m-duck { margin-bottom: 1rem }
-  .m-down { margin-bottom: 2rem }
-  :link,
-  :visited { background-color: inherit; color: inherit }
+
+  img, svg { max-width: 100%; height: auto }
+  :link, :visited { background-color: inherit; color: inherit }
   :focus { outline-color: currentColor }
-  h1 a, h2 a, h3 a, h4 a, h5 a, h6 a { text-decoration:none; }
-  a:hover { text-decoration:underline }
-  @media (min-width:30em) {
-    .site-title { font-size:2.618em; }
-  }
 
-  /* see @link css3base.com/css/fit.css */
-  img { max-width:100%; height:auto; }
-  @media \0screen { img { width: auto } }
```

Comment:
[Hack was for IE8](https://github.com/ryanve/fit/commit/2ee898d81bfa3adc30477db3949a064b3ebf83f0#diff-c67fc19a5a18ea373375e9557ac8f9e7R32)